docs/pages/management/dynamic-resources/user-and-role.mdx

```diff
@@ -0,0 +1,501 @@
+---
+title: Managing users and roles with IaC
+description: Use Infrastructure-as-Code tooling to create Teleport users and roles.
+---
+
+In this guide, you will see how to create users and grant them roles through
+Infrastructure-as-Code (IaC). Teleports supports three ways to dynamically
+create resources from code:
+
+- The Teleport Kubernetes Operator that allows you to manage Teleport resources
+  from Kubernetes
+- The Teleport Terraform Provider that allows you to manage Teleport Resources
+  via Terraform
+- The `tctl` CLI that allows you to manage Teleport resources from your local
+  computer or your CI environment
+
+### Prerequisites
+
+To follow this guide, you must have:
+
+<Tabs>
+<TabItem label="tctl">
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+</TabItem>
+<TabItem label="operator">
+
+A running operator by following either:
+- [the guide to enable the operator in the `teleport-cluster` Helm chart](./teleport-operator-helm.mdx).
+- [the guide to setup a standalone operator](./teleport-operator-standalone.mdx).
+
+You must also set the namespace in which you deployed the operator as this is
+the namespace where you will deploy the CustomResources:
+
+```code
+# for operators deployed with the `teleport-cluster` Helm chart
+$ export OPERATOR_NAMESPACE="teleport-cluster"
+
+# for standalone operators
+$ export OPERATOR_NAMESPACE="teleport-iac"
+```
+
+</TabItem>
+<TabItem label="terraform">
+
+A functional Teleport Terraform provider by following [the Terraform provider guide](./terraform-provider.mdx).
+
+</TabItem>
+</Tabs>
+
+### Step 1/4 - Write manifests
```

Comment:
"Step" sections use the "Step n/d. " format. (This applies to other "Step" sections on this page.)